cypress/integration/models/applicationinventory/applicationinventory.ts

```diff
@@ -15,6 +19,16 @@ import {
     applicationCommentInput,
     editButton,
     actionButton,
+    selectBox,
+    stakeholderSelect,
+    stakeholdergroupsSelect,
+    selectInput,
+    criticalityInput,
+    priorityInput,
+    questionBlock,
+    radioInput,
+    assessmentColumnSelector,
+    reviewColumnSelector,
```

Comment:
Can we have different views - **assessment.view** and **review.view** for these widgets?

Reply:
Done.